src/main/modules/session/index.ts

```diff
@@ -14,6 +14,7 @@ export const cookieMaxAge = 21 * (60 * 1000); // 21 minutes
 export class SessionStorage {
   public enableFor(app: Application): void {
     app.use(cookieParser());
+    //app.set('trust proxy', 1);
```

Comment:
This is required

Reply:
Commented code has been removed as this is already present in server.ts